### PR TITLE
[compute] Compute cli updates

### LINKED ISSUE #8402: az vm create should have a parameter to set osprofile.computerName (hostname)
**Is your feature request related to a problem? Please describe.**
Yes. I can create a VM from one of my images easily for everything including networking but for some reason there isn't a parameter to set osprofile.computerName (ie the hostname) of the new VM.
I need this almost everyday and I have to resort to a group deployment with templates and many files just to set up a simple VM 

**Describe the solution you'd like**
there should be a --computer-name switch to specify the hostname of each created VM.

**Describe alternatives you've considered**
I have to go through the hassle of a az group deployment with JSON files for this simple thing.

**Additional context**

### PR DESCRIPTION
**Pull request by adewaleo**
Closes #8402 #8669 #9026 #5292 

* vm create: Added --computer-name for setting a vm's computer name.
* vm/vmss create: `--ssh-key-value` renamed to `--ssh-key-values` and can now accept multiple ssh public key values or paths.
* Minor doc updates, such as removing "preview" from some help text.
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py [hunk 1]
```diff
@@ -160,6 +160,7 @@ def load_arguments(self, _):
         c.argument('vm_name', existing_vm_name)
         c.argument('size', completer=get_vm_size_completion_list)
         c.argument('name', arg_type=name_arg_type)
+        c.argument('computer_name')
```
Comment (tjprescott) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py]: Why is this here? It is doing nothing.

File: src/command_modules/azure-cli-vm/HISTORY.rst [hunk 2]
```diff
@@ -6,6 +6,7 @@ Release History
 2.2.21
 ++++++
 * vm create: Added --computer-name for setting a vm's computer name.
+* vm/vmss create: `--ssh-key-value` can now accept multiple ssh public key values or paths.
```
Comment (tjprescott) [inline src/command_modules/azure-cli-vm/HISTORY.rst]: Then shouldn't it be `--ssh-key-values`?  This would not be a breaking change due to the way argparse parses things.

Comment (adewaleo) [reply src/command_modules/azure-cli-vm/HISTORY.rst]: Yeah, I didn't want cause confusion, even though it would work. but that concern was probably unnecessary.

**Comment by adewaleo** [comment]
@tjprescott, let me know if you have any other review comments

**Comment by adewaleo** [comment]
Does this look good to you @tjprescott ? I considered renaming namespace.ssh_key_value to namespace.ssh_key_values, what do
You think?
